ftw/faqblock/upgrades/20200723135311_plone_5_upgrade/registry.xml

```diff
@@ -0,0 +1,14 @@
+<?xml version="1.0"?>
+<registry>
+
+    <records prefix="plone.bundles/ftw-faqblock-resources" interface='Products.CMFPlone.interfaces.IBundleRegistry'>
+        <value key="enabled">True</value>
+        <value key="depends">plone</value>
+        <value key="stub_js_modules">jquery</value>
+        <value key="compile">False</value>
+        <value key="jscompilation">++resource++ftw.faqblock/js/faqblock-compiled.js</value>
+        <value key="merge_with">default</value>
+        <value key="last_compilation">2020-06-26 00:42:00</value>
```

Comment:
...same here. If you care, you might want to correct the date.

Reply:
Same as above